sorc/grid_tools.fd/regional_esg_grid.fd/psym2.f90

```diff
@@ -1,5 +1,6 @@
 !> @file
-!! @author R. J. Purser @date September 2018
+!! @author R. J. Purser 
+!! @date September 2018
```

Comment:
We need a blank line here, so there are two doxygen docblocks, one for the file, one for the module.